pkgs/desktops/gnome-3/3.24/misc/gnome-tweak-tool/default.nix

```diff
@@ -15,14 +13,17 @@ in stdenv.mkDerivation rec {
   makeFlags = [ "DESTDIR=/" ];
 
   buildInputs = [ pkgconfig gtk3 glib intltool itstool libxml2
-                  gnome3.gsettings_desktop_schemas makeWrapper file
+                  gnome3.gsettings_desktop_schemas file
                   gdk_pixbuf gnome3.defaultIconTheme librsvg
-                  libnotify gnome3.gnome_shell
+                  libnotify gnome3.gnome_shell python2Packages.pygobject3
                   libsoup gnome3.gnome_settings_daemon gnome3.nautilus
-                  gnome3.gnome_desktop wrapGAppsHook
-                  python2Packages.pygobject3.dev pythonEnv gobjectIntrospection ];
+                  gnome3.gnome_desktop wrapGAppsHook gobjectIntrospection
+                ];
 
-  PYTHONPATH = "$out/${pythonEnv.python.sitePackages}";
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix PYTHONPATH : "$out/${python2Packages.python.sitePackages}")
```

Comment:
Thanks. I made the change but I'm a few hours of compilation away from confirming whether it works. Will update PR with comments when tested.